src/openrct2/interface/Window.cpp

```diff
@@ -952,7 +952,7 @@ static void call_event_viewport_rotate_on_all_windows()
  *      1: clockwise
  *      -1: anti-clockwise
  */
-void window_rotate_camera(rct_window* w, int32_t direction)
+void window_rotate_camera(rct_window* w, Direction direction)
```

Comment:
I don’t like the change here. The argument is a rotation, not one of the four directions. Having to call this with Direction::Invalid (or any of the other names) just looks wrong to me.